frontend/public/index.html

```diff
@@ -11,7 +11,7 @@
   </head>
   <body>
     <noscript>
-      <strong>Sorry, eCamp funktioniert nicht ohne JavaScript. Bitte aktiviere JavaScript um fortzufahren.</strong>
+      <strong></strong>
```

Comment:
Bitte wiederherstellen
```suggestion
      <strong>Sorry, eCamp funktioniert nicht ohne JavaScript. Bitte aktiviere JavaScript um fortzufahren.</strong>
```